libraries/matchers.rb

```diff
@@ -1,9 +1,15 @@
 # encoding: UTF-8
 if defined?(ChefSpec)
```

Comment:
Can we have a preamble explaining what ChefSpec is, with a link to the repo etc? Mainly in interest of future contributors.